nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -459,7 +466,7 @@ define([
         function renderLayout() {
             const events = Events.make(),
                 content = div({
-                    class: `${cssCellType}__layout_container kbase-extension kb-app-cell`,
```

Comment:
Having the `kb-app-cell` class in there was causing some spacing problems and introducing a blue border that's not inline with the designs. The rest of the required styling is in the `kb-bulk-import__layout_container` class, so that seems superfluous anyway.

There's a couple of minor things to be tweaked to the widget container, I think, so I'll do that in another PR.

Reply:
Can you record these todo items in the cleanup ticket in the backlog, or somewhere else?

Reply:
Done, added here: https://kbase-jira.atlassian.net/browse/DATAUP-317